### PR TITLE
Create 2 more tests to test body.json endponts

### PR DESCRIPTION
**Pull request by sacheu**
* Add test for getEmptyFile and getFileLarge
* check the data length in the http response in getFile
* remove a plist file checked in by mistake
* update .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -303,5 +303,5 @@ AutorestSwift.xcodeproj/xcuserdata/
 Pods/
 .build/
 .swiftpm/
-test/integration/generated/body-file/AutoRestSwaggerBatFile.xcodeproj/xcuserdata
-test/integration/generated/head/AutoRestHeadTest.xcodeproj/xcuserdata
+*.plist
```
Comment (tjprescott) [inline .gitignore:306]: I don't think we can just add plists to .gitignore. Some plists we need checked in because each project looks for an info.plist in its build settings.

File: .gitignore [hunk 2]
```diff
@@ -303,5 +303,5 @@ AutorestSwift.xcodeproj/xcuserdata/
...
+test/integration/generated/**/**/xcuserdata
```
Comment (tjprescott) [inline .gitignore:307]: 😍  
Thank you for this!

File: scripts/testserver_codegen.sh [hunk 3]
```diff
@@ -10,6 +10,5 @@ do
     autorest --input-file=./node_modules/@microsoft.azure/autorest.testserver/swagger/$i.json --output-folder=./test/integration/generated/$i --namespace=$i --use=.
     cd ./test/integration/generated/$i
     swift build
-    swift package generate-xcodeproj
     cd ../../../..
```
Comment (tjprescott) [inline scripts/testserver_codegen.sh:13]: This can put you in a wonky state if the `testserver_codegen.sh` script fails. Is there any way to ensure we get back to the original starting directory?

File: src/AutorestSwift/View Models/OperationViewModel.swift [hunk 4]
```diff
@@ -157,7 +157,9 @@ struct OperationViewModel {
             statusCodes.append(contentsOf: response.statusCodes)
         }
         for exception in exceptions {
-            statusCodes.append(contentsOf: exception.statusCodes)
+            for statusCode in exception.statusCodes where statusCode != "default" {
+                statusCodes.append(statusCode)
+            }
```
Comment (tjprescott) [inline src/AutorestSwift/View Models/OperationViewModel.swift:162]: Is `exceptions` a model object or a view model object? If it's a model object, it seems we could have a calculated property or helper method to handle this logic (I've seen it repeated, I think). If it's a view model, then I think we are already making sure that status codes is only numeric.

File: templates/OperationResponseFailureSnippet.stencil [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{% if op.defaultExceptionHasBody == true %}
+{% include "OperationResponseFailureBodySnippet.stencil" %}
+{% else %}
+{% include "OperationResponseFailureNoBodySnippet.stencil" %}
+{% endif %}
```
Comment (tjprescott) [inline templates/OperationResponseFailureSnippet.stencil:5]: Wasn't this already checked in?

File: scripts/testserver_codegen.sh [hunk 6]
```diff
@@ -10,6 +10,5 @@ do
     autorest --input-file=./node_modules/@microsoft.azure/autorest.testserver/swagger/$i.json --output-folder=./test/integration/generated/$i --namespace=$i --use=.
     cd ./test/integration/generated/$i
     swift build
-    swift package generate-xcodeproj
```
Comment (tjprescott) [inline scripts/testserver_codegen.sh:13]: Why was this line removed?

**Review by tjprescott — CHANGES_REQUESTED** [review]
Mostly questions.

File: .gitignore (hunk 1 above)
Comment (sacheu) [reply .gitignore:306]: Thanks. Revert.

File: templates/OperationResponseFailureSnippet.stencil (hunk 5 above)
Comment (sacheu) [reply templates/OperationResponseFailureSnippet.stencil:5]: Sorry this should be removed

File: scripts/testserver_codegen.sh (hunk 3 above)
Comment (sacheu) [reply scripts/testserver_codegen.sh:13]: I need to investigate how we can do that. Will be address in other PR.

File: scripts/testserver_codegen.sh (hunk 6 above)
Comment (sacheu) [reply scripts/testserver_codegen.sh:13]: i don't want to recreate a new Xcode project for the generated code everytime because we need to modify the generated XCode project before we can add to AutorestswiftTest xcode project. And i don't want the manual changed to be overwritten everytime we run this script. The modification we need to make in the generated Xcode project is to remove the AzureCore target in each of the generated Xcode project. Without this modification, Xcode will complain saying there are multiple AzureCore targets (since each Xcode project will have its own AxureCore target by default (without the manually deletion)

**Review by tjprescott — APPROVED** [review]
